src/control/provider/system/distribution.go

```diff
@@ -118,7 +118,6 @@ func getKernelVersion(kernel *KernelVersion, open openFunc) {
 				break
 			}
 		}
-		break
```

Comment:
This change potentially changes the functionality, doesn't it? If there are multiple lines, it could read them. If we only want to read the first line, maybe the `for scn.Scan()` should be `if scn.Scan()`.

Reply:
yes good point, will change to "if"